src/GraphQL.Tests/Types/AutoRegisteringObjectGraphTypeTests.cs

```diff
@@ -386,6 +442,24 @@ public override void Modify(ArgumentInformation argumentInformation)
                 => argumentInformation.SetDelegate(context => 85);
         }
 
+        private class NoDefaultConstructorTest
+        {
+            public NoDefaultConstructorTest(bool value)
+            {
+                Example1 = value;
+            }
+
+            public bool Example1 { get; set; }
+            public string Example2() => "test";
+            public int Example3 = 1;
+        }
+
+        private class NullSourceTest
+        {
+            public static bool Example1 { get; set; } = true;
+            public static string Example2() => "test";
```

Comment:
Would like to add  `public static int Example3 = 1;` field and test for it

Reply:
Done